tests/pcmsolver/opt-fd/input.dat

```diff
@@ -0,0 +1,42 @@
+#! PCM-SCF STO-3G geometry optimization by finite differences, with Z-matrix input
+# Reference values from DALTON (inputs and output included)
+nucenergy = 8.924418385165        #TEST
+refenergy = -74.971455946174      #TEST
+
+
+molecule h2o {
+  symmetry c1
+  O
+  H 1 1.0
+  H 1 1.0 2 104.5
+}
+
+set {
+  basis sto-3g
+  e_convergence 10
+  d_convergence 10
+  scf_type pk
+  pcm true
+  pcm_scf_type total
+}
+
+pcm = {
+  Units = Angstrom
+  Medium {
+    SolverType = IEFPCM
+    Solvent = Water
+  }
+
+  Cavity {
+    RadiiSet = UFF
+    Type = GePol
+    Scaling = False
+    Area = 0.3
+    Mode = Implicit
+  }
+}
+
+thisenergy = optimize('scf', molecule=h2o, dertype='energy')
```

Comment:
Not necessary for this PR, but can we get pcmsolver to automatically fall back on finite difference for optimize (as I think ECP does), rather than having the user be explicit?

Reply:
RN updated.